### ISSUE TITLE
Minju: Behaviour, Strategy, Implementation; 3 weeks 

### ISSUE THREAD

**Issue by minjupgeorge** [lab-brussels-1/home#346]

**Comment by minjupgeorge** [comment]
# Week 1
---

## I Need Help With:
- I felt it very difficult to write unit test in sandbox .test.js . I need some more time to master it

## What went well?
- I am very happy  to practice coding at Edabit website. 
- I understood the need of Testing but still need practice.

## What went less well?

- I am watching the video and trying to understand the testing .

## Lessons Learned

- Needs a lot pf practice and time for understanding if I miss the class.

## Sunday Prep Work

- still trying to pass test cases

**Comment by danielhalasz** [comment]
> I felt it very difficult to write unit test in sandbox .test.js . I need some more time to master it

which part was difficult for you? have you looked at the examples? you can also look at the Jest documentation, which is the tool used at HYF for testing..keep in mind that there are others, with a slightly different syntax on some coding challenge websites.

> I am watching the video and trying to understand the testing .

which video? the class recording? any questions about it?

**Comment by minjupgeorge** [comment]
@danielhalasz  yeah I mean our class video. have tried some examples.. now its fine.  ok sure I will go through Jest documentation.

**Comment by minjupgeorge** [comment]
# Week 2

## I Need Help With:

-I am struggling with one syntax error in node  modules whenever I try to run the test. I had zoom call with Daniel and understood its because of windows laptop. still trying to correct it.

## What went well?

-  write different test cases for solution-write up repo 

 ## What went less well?
****
![image](https://user-images.githubusercontent.com/32439811/177993664-34247083-55c0-4b7c-96fd-75fd5f19b053.png)

The node module file is

![image](https://user-images.githubusercontent.com/32439811/177993719-ebde3a5f-8e2a-4307-b57a-befe0f93e133.png)

 ## Lessons Learned

More practice is needed.

## Sunday Prep Work
- still trying to figure out the error .

**Comment by minjupgeorge** [comment]
@danielhalasz Thank you for the great help and sharing the reference link. Now its working with this code .
node ./node_modules/jest/bin/jest.js doctor-example.test.js
 This link also very helpful. 
https://github.com/facebook/jest/issues/3750#issuecomment-308118403

**Comment by minjupgeorge** [comment]
@danielhalasz Thank you for the great help and sharing the reference link. Now its working with this code .
node ./node_modules/jest/bin/jest.js doctor-example.test.js
 This link also very helpful. 
https://github.com/facebook/jest/issues/3750#issuecomment-308118403
